e2e/terraform/config/full-cluster/vault/server/vault.hcl

```diff
@@ -1,12 +1,15 @@
-backend "consul" {
-  path          = "vault/"
-  address       = "{{ GetPrivateIP }}:8500"
-  cluster_addr  = "https://{{ GetPrivateIP }}:8201"
-  redirect_addr = "http://{{ GetPrivateIP }}:8200"
+listener "tcp" {
+  address     = "0.0.0.0:8200"
+  tls_disable = 1
 }
 
-listener "tcp" {
-  address         = "{{ GetPrivateIP }}:8200"
-  cluster_address = "{{ GetPrivateIP }}:8201"
-  tls_disable     = 1
+# this autounseal key is created by Terraform in the E2E infrastructure repo
+# and should be used only for these tests
+seal "awskms" {
+  region     = "us-east-1"
+  kms_key_id = "74b7e226-c745-4ddd-9b7f-2371024ee37d"
 }
+
+# Vault doesn't have autodiscovery for retry_join on its
```

Comment:
```suggestion
# Vault 1.5.4 doesn't have autodiscovery for retry_join on its
```
I'm sure the storage will stay as consul for awhile, but maybe including the version number will motivate a future reader to see if vault is ready

Reply:
Good call